File: Arrays101/Deleting_item_from_array/RemoveElement.java
package Deleting_item_from_array;
public class RemoveElement {
    public static void main(String[] args) {
        int [] nums = {3,2,2,3};

        removeElement(nums, 3);
    }

    public static int removeElement(int[] nums, int val) {
        int count = 0;

        for (int i = 0; i < nums.length; i++) {
            if(nums[i] != val){
                nums[count++] = nums[i];
            }
        }

        while(count < nums.length){
            nums[count++] = val;
        }


        return count;
    }

   

}

File: Arrays101/Deleting Item From Array/removeElement.py
def removeElement(nums, val):
    # Initialize a pointer for the position of the next non-val element
    k = 0
    
    # Iterate through the array
    for i in range(len(nums)):
        # If the current element is not equal to val, place it at index k
        if nums[i] != val:
            nums[k] = nums[i]
            k += 1
    
    return k  # k is the count of elements not equal to val
